download_traffic_data.py

```
@@ -0,0 +1,234 @@
+#!/usr/bin/env python3
+import argparse
+import asyncio
+from datetime import datetime, timezone
+from pathlib import Path
+import sys
```

Ruff (0.12.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)